pluma/pluma-prefs-manager-app.c

```diff
@@ -1439,3 +1476,85 @@ pluma_prefs_manager_lockdown_changed (GSettings *settings,
 					     PLUMA_LOCKDOWN_SAVE_TO_DISK,
 					     locked);
 }
+
+static void
+pluma_prefs_manager_draw_generic (GSettings  *settings,
+                                  gchar      *key,
+                                  gchar      *real_key,
+                                  GtkSourceSpaceTypeFlags type)
+{
+        pluma_debug (DEBUG_PREFS);
+
+        if (strcmp (key, real_key) == 0)
+        {
+                gboolean enabled;
+                GList *views;
+                GList *l;
+
+                enabled = g_settings_get_boolean (settings, key);
+
+                views = pluma_app_get_views (pluma_app_get_default ());
+                l = views;
+
+                while (l != NULL)
+                {
+                        GtkSourceSpaceTypeFlags value;
+
+                        value = gtk_source_view_get_draw_spaces (GTK_SOURCE_VIEW (l->data));
+                        if (enabled)
+                                value |= type;
+                        else
+                                value &= ~type;
+                        gtk_source_view_set_draw_spaces (GTK_SOURCE_VIEW (l->data),
+                                                                  value);
+
+                        l = l->next;
+                }
+
+                g_list_free (views);
+        }
+}
+
+static void
+pluma_prefs_manager_draw_spaces_changed (GSettings *settings,
+                                         gchar       *key,
+                                         gpointer     user_data)
+{
+        pluma_prefs_manager_draw_generic(settings,
```

Comment:
space between function name and opening parenthesis: `...generic (settings`

Reply:
This should still be done, we usually leave a space there
```
pluma_prefs_manager_draw_generic (settings,
```

Reply:
I updated the patch and now there is a space between the function name and '('